pkg/data/helpers.go

```diff
@@ -13,25 +13,27 @@ func getFiltersFromStruct(i interface{}, keyName *string) (*expression.KeyCondit
 	v := reflect.ValueOf(i).Elem()
 	for i := 0; i < v.NumField(); i++ {
 		dField := strings.Split(v.Type().Field(i).Tag.Get("dynamodbav"), ",")[0]
-		value := v.Field(i).Interface()
-		if !reflect.ValueOf(value).IsNil() {
-			switch v.Field(i).Kind() {
-			case reflect.Ptr:
-				dValue := reflect.Indirect(v.Field(i)).Interface()
-				if keyName != nil {
-					if dField == *keyName {
-						newFilter := expression.Key(dField).Equal(expression.Value(dValue))
-						kb = &newFilter
-						continue
+		if dField != "-" {
```

Comment:
Forgot to account on the void tag.  Need this to skip adding queries for limit and nextId